src/k8s-engine/lima.ts

```diff
@@ -1345,12 +1397,26 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
         await this.startVM();
 
         await this.deleteIncompatibleData(isDowngrade);
-        this.lastCommandComment = 'Installing k3s, trivy & CA certs';
-        await Promise.all([
-          this.progressTracker.action('Installing k3s', 50, async() => {
+        if (this.#currentContainerEngine === ContainerEngine.CONTAINERD) {
+          this.lastCommandComment = 'Starting containerd';
+          await this.progressTracker.action(this.lastCommandComment, 50, this.startContainerd());
+        } else if (this.#currentContainerEngine === ContainerEngine.MOBY) {
+          this.lastCommandComment = 'Starting dockerd';
+          await this.progressTracker.action(this.lastCommandComment, 50, async() => {
+            await this.ssh('sudo', '/sbin/rc-service', '--ifnotstarted', 'docker', 'start');
+          });
+        }
```

Comment:
Given that we have dependencies, this only makes sense when k3s is disabled.

Reply:
Right. Same for stopping `docker`.